app/viewModels/taskList/Task.scala

```diff
@@ -36,7 +36,7 @@ abstract class TaskListTask extends Task {
       case Completed | InProgress | Error => s"task.$messageKey.edit"
       case NotStarted                     => s"task.$messageKey.add"
       case CannotStartYet | Unavailable   => s"task.$messageKey"
-      case TaskStatus.Amended             => s"task.$messageKey"
+      case TaskStatus.Amended             => s"task.$messageKey.edit"
```

Comment:
Very minor but could remove this and change line 36 to `case Completed | InProgress | Error | Amended`

Reply:
Is this unit tested?